hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataWriteUtils.java

```diff
@@ -120,6 +121,11 @@ public static HoodieWriteConfig createMetadataWriteConfig(
             .withMaxConsistencyChecks(writeConfig.getConsistencyGuardConfig().getMaxConsistencyChecks())
             .build())
         .withWriteConcurrencyMode(WriteConcurrencyMode.SINGLE_WRITER)
+        .withFileSystemViewConfig(FileSystemViewStorageConfig.newBuilder()
+            .withStorageType(writeConfig.getMetadataConfig().getMetadataViewType())
+            .withBaseStoreDir(writeConfig.getViewStorageConfig().getSpillableDir())
```

Comment:
should we suffix "/metadata" to writeConfig.getViewStorageConfig().getSpillableDir() 
so that we de-couple data table and metadata table spillable dir. 
or you intentionally wanted to keep it to just 1 dir for a given table.